create-meta/java/src/test/java/io/cucumber/createmeta/CleanSensitiveInformationTest.java

```diff
@@ -0,0 +1,33 @@
+package io.cucumber.createmeta;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.HashMap;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.matchesPattern;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class CleanSensitiveInformationTest {
```

Comment:
With JUnit 5 test classes can be package private. This is nice because code completion won't suggest them outside of their package.